GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -176,17 +177,15 @@ void Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat,
 }
 
 // generate vertex array object
-void Mesh::generateVAO(Material::ShaderType key) {
+void Mesh::generateVAO() {
 #if _GVRF_USE_GLES3_
+
+	if (vaoInitiliased_)
+		return;
```

Comment:
These two lines are tab based formatting not space based. 
Please in Window -> Preference -> C/C++ -> Code Style
And then edit the K&R[Space] profile, change the profile name and for Tab policy, change it from tabs only to spaces only.

Then you do Ctrl + Shift + F, you will see that there is something changed.

Thanks in advance.